server/Controllers/Mention.Controller.js

```diff
@@ -0,0 +1,110 @@
+const Mention = require("../models/Mention");
+const { mentionValidation } = require("../utils/validation");
+
+module.exports = {
+  getAllMentions: async (req, res) => {
```

Comment:
This will need to be updated to get the mentions for a given company, but that can be added once login is done